net/http.ts

```diff
@@ -323,3 +324,13 @@ async function readAllIterator(
   }
   return collected;
 }
+
+const http = {
+  Handle: function() {
+
+  },
+  FileServer,
+};
+
+export default http;
```

Comment:
I think users can import directly from `file_serve.ts` instead of `http.ts`.

Reply:
AFAICT we prefer named exports over default ones.

Reply:
Yes - let’s avoid default exports.